recipes-extended/xen/files/shim-support-for-shim-lock-measure.patch

```diff
@@ -73,12 +43,19 @@ Signed-off-by: Tamas K Lengyel <lengyelt@ainfosec.com>
      union string section = { NULL }, name;
      bool base_video = false;
 @@ -1225,6 +1260,13 @@ efi_start(EFI_HANDLE ImageHandle, EFI_SY
```

Comment:
`@@ -1225,6 +1260,13` should be `@@ -1187,6 +1196,8 @@` for the patch to apply.
The patch may use a refresh :).

Reply:
Whoops, thx